c/bpxskt.c

```diff
@@ -1015,7 +1025,9 @@ int socketRead(Socket *socket, char *buffer, int desiredBytes,
           returnCode,
           reasonCodePtr);
   if (returnValue < 0){
-    printf("read failed ret code %d reason 0x%x\n",*returnCode,*reasonCode);
+    if (socketTrace){
+      printf("read failed ret code %d reason 0x%x\n",*returnCode,*reasonCode);
```

Comment:
i think this is not needed because returnvalue, returncode, and reasoncode seem to get back to caller.